homeassistant/components/denonavr/__init__.py

```diff
@@ -53,6 +55,9 @@ async def async_setup_entry(
 
     hass.data[DOMAIN][entry.entry_id] = {
         CONF_RECEIVER: receiver,
+        CONF_UPDATE_AUDYSSEY: entry.options.get(
```

Comment:
We don't need to store entry options in hass.data. We have access to the entry in the media player platform.